From 0369e2fde4ae2de7cbc1b426c3eb9e6fcb2c5a32 Mon Sep 17 00:00:00 2001
From: Tom Rondeau <tom@trondeau.com>
Date: Wed, 10 Feb 2016 12:14:37 -0500
Subject: [PATCH 01/19] cmake: Fixed bug in FindThrift.cmake

pkg-config sets the variable INCLUDE_DIRS, not INCLUDE_DIR. Cmake
wouldn't pick up the thrift package if installed in another prefix
without this.
---
 cmake/Modules/FindThrift.cmake | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/cmake/Modules/FindThrift.cmake b/cmake/Modules/FindThrift.cmake
index cef8a47..4702f90 100644
--- a/cmake/Modules/FindThrift.cmake
+++ b/cmake/Modules/FindThrift.cmake
@@ -13,10 +13,9 @@ endif(PC_THRIFT_FOUND AND PC_THRIFT_VERSION VERSION_LESS ${THRIFT_REQ_VERSION})
 
 
 # Else, look for it ourselves
-
 FIND_PATH(THRIFT_INCLUDE_DIRS
   NAMES thrift/Thrift.h
-  HINTS ${PC_THRIFT_INCLUDE_DIR}
+  HINTS ${PC_THRIFT_INCLUDE_DIRS}
   ${CMAKE_INSTALL_PREFIX}/include
   PATHS
   /usr/local/include
-- 
2.1.4

